projects/common/src/time/time-range.service.test.ts

```diff
@@ -78,10 +78,7 @@ describe('Time range(TR) service', () => {
                 }
               } as unknown) as ActivatedRoute
             }),
-            getQueryParameter: jest
-              .fn()
-              .mockReturnValueOnce(firstArrivingTimeRange.toUrlString())
-              .mockReturnValue(secondArrivingTimeRange.toUrlString()),
+            getQueryParameter: jest.fn().mockReturnValue(secondArrivingTimeRange.toUrlString()),
```

Comment:
this function is only used to determine changes which have their own subscriber now, so the first nav doesn't trigger it.